src/sentry/utils/snuba.py

```diff
@@ -546,6 +550,16 @@ def transform_aliases_and_query(**kwargs):
 
     result = dataset_query(**kwargs)
 
+    return transform_results(result, translated_columns, kwargs)
+
+
+def transform_results(result, translated_columns, snuba_args):
```

Comment:
nit: Where is the future home of this function? Should it live in the Discover module now together with the new new transformation code that goes from public -> internal? Since the one in this file is now deprecated.